docs/UserGuide.md

```diff
@@ -136,6 +136,36 @@ Examples:
 * `list` followed by `delete 2` deletes the 2nd patient in the address book.
 * `find Betsy` followed by `delete 1` deletes the 1st patient in the results of the `find` command.
 
+### Adding an appointment: `assign`
+
+Assign the specified patient into the specified appointment date and time.
+
+Format: `assign INDEX d/DATE t/TIME`
+
+* Puts the person at the specified INDEX into an appointment time slot.
+* The INDEX refers to the index number indicated in the patient list.
+* The IC **must be a positive integer** 1, 2, 3, …​
+* The `DATE` and `TIME` of the appointment must be included.
+* The timeslot indicated by `DATE` and `TIME` must be available.
+* The specified `DATE` and `TIME` must be in the future.
+
+Examples:
+* `assign 1 d/Sunday t/2am` books an appointment at the upcoming Sunday, 2am for the 1st patient in the list.
```

Comment:
Perhaps we can consider including another section detailing our acceptance of natural date syntax above to provide users with a clearer understanding of our date syntax.

Reply:
Hmmm, do you think if something like that will be good? [See](https://jinhao-l.github.io/ip/#3-date-time-support)

Reply:
Yup! It looks great :)